### ISSUE TITLE
Tink migration to Java 11 is showing a seriously Java-warning (illegal reflective access)

### ISSUE THREAD

**Issue by java-crypto** [google/tink#176]
Hi team,
I tested my Tink-routines with Java 11 and a red written warning was shown even with the second step of an encryption setup:

```
TinkConfig.register();
// create keyset
KeysetHandle keysetHandle = KeysetHandle.generateNew(AeadKeyTemplates.AES128_GCM);
```

Create Google Tink KeysetHandle with Java 11
**WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.protobuf.UnsafeUtil (file:/S:/.../protobuf-java-3.6.1.jar) to field java.nio.Buffer.address
WARNING: Please consider reporting this to the maintainers of com.google.protobuf.UnsafeUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release**

As Java 12 is actual in beta pahse there is a possibility that protobuf will no longer work and that would effect Tink as well.

What are your plans to avoid the problem (illegal reflective access caused by protobuffer)?

The minimal sourcecode is available on my Github-Depot (https://github.com/java-crypto/TinkTest/blob/master/CreateKeysetHandleJava11V1.java) and is attached.

Kind regards,
Michael

[CreateKeysetHandleJava11V1.zip](https://github.com/google/tink/files/2778690/CreateKeysetHandleJava11V1.zip)

**Comment by tholenst** [comment]
This is https://github.com/protocolbuffers/protobuf/issues/3781. It should be fixed with the next release of protobuf.

If you want, you can check with master; otherwise I suggest to leave this open until a new version of protobuf is released which includes this fix (the newest one seems to be 3.6.1.3 which doesn't seem to have it yet, see https://github.com/protocolbuffers/protobuf/releases).

**Comment by plumstar** [comment]
> 
> 
> This is [protocolbuffers/protobuf#3781](https://github.com/protocolbuffers/protobuf/issues/3781). It should be fixed with the next release of protobuf.
> 
> If you want, you can check with master; otherwise I suggest to leave this open until a new version of protobuf is released which includes this fix (the newest one seems to be 3.6.1.3 which doesn't seem to have it yet, see https://github.com/protocolbuffers/protobuf/releases).

It seems no fix in the latest version 3.6.1.3.  Please resolve this problem quickly, thans.

**Comment by tholenst** [comment]
Indeed, the fix isn't in 3.6.1.3, it's only in the head of protobuf master. 

If you need a release of protobuf with this included, you need to comment on protocolbuffers/protobuf#3781: the Tink team cannot release a new protobuf version.

**Comment by JnRouvignac** [comment]
Hi!

This warning has now been fixed in protocol buffers v3.7.0 ; now v3.7.1 is even available.
If you update the library, then you can close this issue :)

**Comment by java-crypto** [comment]
Thanks for your information. I checked the new version 3.7.1 with Java 11 and the warning is gone, thanks to all of you.
Kind regards, Michael
